app/src/main/java/com/tramsun/donations/LoginActivity.kt

```diff
@@ -45,6 +46,17 @@ class LoginActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_login)
 
+        //Instantiate different Timber logging trees based on Debug or Release modes
+        if (BuildConfig.DEBUG) {
+            Timber.plant(object : Timber.DebugTree() {
```

Comment:
We can just use DebugTree() and we don't need to override createStackElementTag method as this functionality i built in internally